app/utils/appExclusionsManager.js

```diff
@@ -0,0 +1,66 @@
+const EventEmitter = require('events')
+const log = require('electron-log')
+const psList = require('ps-list')
+
+class AppExclusionsManager extends EventEmitter {
+  constructor (settings) {
+    super()
+    this.timer = null
+    this.appExclusion = settings.get('appExclusions').find(ex => ex.active)
```

Comment:
find() would probably return only one item, though presumably several active ones might be defined.

Reply:
Ah, disregard that, it's intended behavior. Sorry for the noise.